src/test.python.bridge.cpp

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+#include <memory>
+
+#include <Python.h>
+
+#include <openassetio/log/LoggerInterface.hpp>
+#include <openassetio/hostApi/ManagerImplementationFactoryInterface.hpp>
+#include <openassetio/python/hostApi.hpp>
+
+namespace {
+struct DummyLoggerInterface : openassetio::log::LoggerInterface {
+  void log(Severity severity, const openassetio::Str& message) override {}
+};
+}  // namespace
+
+int main() {
+  Py_Initialize();
+  auto logger = std::make_shared<DummyLoggerInterface>();
+  {
+    const auto factory =
+        openassetio::python::hostApi::createPythonPluginSystemManagerImplementationFactory(logger);
```

Comment:
A mouthful.... just saying. You'll be competing with Katana soon.

Reply:
:laughing: Yeah we had great fun coming up with this name. Hardest problem in software engineering, naming things.